tests/run/methodmangling_T5.py

```diff
@@ -1,6 +1,8 @@
 # mode: run
 # ticket: 5
 
+# Some more tests for the same sort of thing are in "methodmangling_T1382"
```

Comment:
"… and the reason why there are two test files is …"
(It's nice to not have to think too much about where a new test should go.)

Reply:
The bulk of the tests are now in `methodmangling_T5.py`. There's still an extra "pure" and "cdef" file to test specific features that don't fit in the main file, but they're small and with a well-defined purpose.